test/lint/lint-circular-dependencies.sh

```diff
@@ -87,10 +87,11 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "evo/deterministicmns -> validationinterface -> evo/deterministicmns"
     "logging -> util/system -> sync -> logging/timer -> logging"
 
+    "coinjoin/client -> net_processing -> coinjoin/client"
```

Comment:
can you avoid adding new circulat dependencies?

These 2 disappeared just because they have shorter path between coinjoin/client.... coinjoin/client

```
"coinjoin/client -> coinjoin/util -> wallet/wallet -> psbt -> node/transaction -> node/context -> coinjoin/context -> coinjoin/client"
"coinjoin/client -> coinjoin/coinjoin -> llmq/chainlocks -> net_processing -> coinjoin/client"
```